### PR TITLE
[zh-cn]sync kubeadm-init api-concepts

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md
content/zh-cn/docs/reference/using-api/api-concepts.md

### CODE REVIEW

File: content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 1]
```diff
@@ -56,14 +56,14 @@ following steps:
...
+1. Writes kubeconfig files in `/etc/kubernetes/` for the kubelet, the controller-manager and the
+   scheduler to use to connect to the API server, each with its own identity. Also
+   additional kubeconfig files are written, for kubeadm as administrative entity (`admin.conf`)
+   and for a super admin user that can bypass RBAC (`super-admin.conf`).
 -->
 3. 将 kubeconfig 文件写入 `/etc/kubernetes/` 目录以便 kubelet、控制器管理器和调度器用来连接到
-   API 服务器，它们每一个都有自己的身份标识，同时生成一个名为 `admin.conf` 的独立的 kubeconfig
-   文件，用于管理操作。
+   API 服务器。此外还编写了额外的 kubeconfig 文件，将 kubeadm 作为管理实体（`admin.conf`）和可以绕过
+   RBAC 的超级管理员用户（`super-admin.conf`）。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md:66]:
```suggestion
   API 服务器，它们每一个都有自己的身份标识。再编写额外的 kubeconfig 文件，将 kubeadm
   作为管理实体（`admin.conf`）和可以绕过 RBAC 的超级管理员用户（`super-admin.conf`）。
```

**Review by windsonsea — COMMENTED** [review]
/lgtm

File: content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 2]
```diff
@@ -429,17 +429,31 @@ List of removed feature gates:
...
+-->
+特性门控的描述：
+
+<!--
+`IPv6DualStack`
+: This flag helps to configure components dual stack when the feature is in progress. For more details on Kubernetes
+dual-stack support see [Dual-stack support with kubeadm](/docs/setup/production-environment/tools/kubeadm/dual-stack-support/).
+-->
+`IPv6DualStack`
+: 当前此特性正在推进时，此标志有助于配置组件的 IP 双栈。有关 Kubernetes
```
Comment (tengqm) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md:454]:
```suggestion
: 在 IP 双栈特性处于开发过程中时，此标志有助于配置组件的双栈支持。有关 Kubernetes
```

File: content/zh-cn/docs/reference/using-api/api-concepts.md [hunk 3]
```diff
@@ -538,15 +538,14 @@ response (10-20MB) and consume a large amount of server resources.
...
-Kubernetes API server supports the ability to break a single large collection request
+The Kubernetes API server supports the ability to break a single large collection request
 into many smaller chunks while preserving the consistency of the total request. Each
 chunk can be returned sequentially which reduces both the total size of the request and
 allows user-oriented clients to display results incrementally to improve responsiveness.
 -->
-如果你没有明确禁用 `APIListChunking` [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)，
-Kubernetes API 服务器支持将单个大型集合请求分解为许多较小块的能力，同时保持总请求的一致性。
+Kubernetes API 服务器支持将单个大型集合请求分解为许多较小块的能力，
+同时保持总请求的一致性。每个块都可以按顺序返回，这既减少了请求的总大小，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/using-api/api-concepts.md:547]:
```suggestion
同时保持总体请求的一致性。每个块都可以按顺序返回，这既减少了请求的总大小，
```

**Pull request merged**